src/KubernetesClient.Models/ResourceQuantity.cs

```diff
@@ -57,7 +54,7 @@ namespace k8s.Models
     ///     cause implementors to also use a fixed point implementation.
     /// </summary>
     [JsonConverter(typeof(QuantityConverter))]
-    public partial class ResourceQuantity : IYamlConvertible
```

Comment:
I'm worried about dropping YAML support here? Is that actually what is happening?

Reply:
Nevermind, I see we added a converter below.